forJIS7/src/main/java/JavaHomework/Lectures_9/Interfaces/Animal/Bird.java

```diff
@@ -0,0 +1,56 @@
+package JavaHomework.Lectures_9.Interfaces.Animal;
+
+import java.util.Objects;
+
+public class Bird extends Animals {
+
+    private String birdName;
```

Comment:
You dont need this field here because Animals has this field

Reply:
fixed